### PR TITLE
3 small cleanups that I don't intend to squash but rebase :)

### PR DESCRIPTION
**Pull request by lacostej**

### CODE REVIEW

File: fastlane/actions/test_sample_code.rb [hunk 1]
```diff
@@ -95,10 +95,8 @@ def self.is_supported?(platform)
       end
 
       def self.fill_in_env_variables
-        # Some code samples need a value in a certain env variable
-        ["GITHUB_TOKEN"].each do |current|
```
Comment (KrauseFx) [inline fastlane/actions/test_sample_code.rb]: Why not just leave the `each` and add the `CI` parameter here? It's nice to make it easy to add further parameters.

Comment (lacostej) [inline fastlane/actions/test_sample_code.rb]: I don't get how that would help
```ruby
["GITHUB_TOKEN", "CI"].each do |current|
  # what do I do with current here?
end
```

**Comment by KrauseFx** [comment]
👍

**Comment by joshdholtz** [comment]
@lacostej Decided to go through all docs PRs today and saw this one was still open but looks to be over a year out of date 😬 Would you like to rebase this PR based on current docs? Or should we close?

**Comment by KrauseFx** [comment]
Let's close this for now 👍

**Pull request closed without merging**